delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeIndexingStream.scala

```diff
@@ -96,20 +104,41 @@ final class CompositeIndexingStream(
       }
       .flatMap { progressMap =>
         val streams = view.value.sources.value.map { source =>
-          val sourcePId = sourceProjection(source, view.rev)
-          val progress  = progressMap(sourcePId)
+          val sourcePId                                                           = sourceProjection(source, view.rev)
+          val progress                                                            = progressMap(sourcePId)
           // fetches the source stream
-          val stream    = source match {
-            case s: ProjectSource       => indexingSource(view.projectRef, progress.offset, s.resourceTag)
-            case s: CrossProjectSource  => indexingSource(s.project, progress.offset, s.resourceTag)
-            case _: RemoteProjectSource => Stream.empty // TODO: To be implemented
+          val stream: Stream[Task, Chunk[Message[BlazegraphIndexingStreamEntry]]] = source match {
+            case s: ProjectSource       =>
+              indexingSource(view.projectRef, progress.offset, s.resourceTag).evalMapValue(
+                BlazegraphIndexingStreamEntry.fromEventExchange(_)
+              )
+            case s: CrossProjectSource  =>
+              indexingSource(s.project, progress.offset, s.resourceTag).evalMapValue(
+                BlazegraphIndexingStreamEntry.fromEventExchange(_)
+              )
+            case s: RemoteProjectSource =>
+              remoteProjectStream(s, progress.offset)
```

Comment:
All this block seems to much here. It was already complicated, now is even more.
All this part, for `ProjectSource` and `CrossProjectSource` is handled in the `IndexingSource`

It should be the same for remote projects. 

The easier way would be to add `CompositeViewIndexingSource` as follows:
```scala
trait CompositeViewIndexingSource {
  def apply(
      project: ProjectRef,
      offset: Offset,
      tag: Option[TagLabel]
  ): Stream[Task, Chunk[Message[BlazegraphIndexingStreamEntry]]]
}
```

And then on the `CompositeIndexingStream` class you can pass `indexingSource: CompositeViewIndexingSource, remoteIndexingSource: CompositeViewIndexingSource` and all this block will be like before:
```scala
            case s: ProjectSource       => indexingSource(view.projectRef, progress.offset, s.resourceTag)
            case s: CrossProjectSource  => indexingSource(s.project, progress.offset, s.resourceTag)
            case _: RemoteProjectSource => remoteIndexingSource(view.projectRef, progress.offset, None)
```